respy/pre_processing/model_processing.py

```diff
@@ -168,6 +169,22 @@ def _parse_inadmissibility_penalty(optim_paras, params):
     return optim_paras
 
 
+def _parse_exogenous_processes(optim_paras, params):
```

Comment:
While I like how this extension fits into the existing logic of the model specification I think we should keep in mind that the params file is growing larger and more complex as one adds features to the model. 
I had the experience that it can be quite tedious to adapt the file in excel or so. Maybe we can think about a way to automatically adapt these specs or to simplify the specification management in the medium run.

Reply:
Great idea. My new `generate_random_model` function is built on the idea that you pass a list of features similar to constraints in estimagic to the function and it builds a model. The underlying functions which generate each feature would be useful for this. What do you have in mind? A GUI which is a little bit excel and has feature generators?